mrmustard/lab_dev/transformations/transformations.py

```diff
@@ -389,6 +389,77 @@ def representation(self) -> Bargmann:
         return Bargmann(*triples.twomode_squeezing_gate_Abc(self.r.value, self.phi.value))
 
 
+class Amplifier(Channel):
+    r"""The noisy amplifier channel.
+
+    If ``gain`` is an iterable, its length must be equal to `1` or `N`. If it length is equal to `1`,
+    all the modes share the same gain.
+
+    .. code-block ::
+
+        >>> import numpy as np
+        >>> from mrmustard.lab_dev import Amplifier
+
+        >>> channel = Amplifier(modes=[1, 2], gain=1.1)
+        >>> assert channel.modes == [1, 2]
+        >>> assert np.allclose(channel.gain.value, [1.1, 1.1])
```

Comment:
please update these with an actual test of some kind
```suggestion
        >>> amp = Amplifier([0], gain=2)
        >>> coh = Coherent([0], x=1.0, y=2.0)  # units of hbar=2 (default) 
        >>> _, mu, _ = (coh >> amp).phase_space(0)
        >>> assert np.allclose(mu[0], np.array([2.0, 4.0]))
```